playbooks/openshift-master/private/set_network_facts.yml

```diff
@@ -17,12 +17,6 @@
   - role: openshift_facts
   post_tasks:
   - block:
-    - set_fact:
```

Comment:
This code needs to be somewhere.  We have some newer slurp commands that we can probably move this into inside of init code.